tests/integration.rs

```diff
@@ -5484,6 +5502,7 @@ mod tests {
         }
 
         #[test]
+        #[cfg(not(feature = "mshv"))]
         fn test_ovs_dpdk() {
```

Comment:
Ditto: given the basic "test_ovs_dpdk" test is not working for MSHV, what about the `test_live_migration_ovs_dpdk`?